packs/classfeatures/avenger.json

```diff
@@ -134,7 +134,7 @@
         },
         "traits": {
             "otherTags": [
-                "rogue-racket"
+                "rogue-class-archetype"
             ],
```

Comment:
Instead, let's make two tags. One for racket and one that's just `class-archetype` then we can restrict the class archetype choices for dedications.